narwhal/primary/src/proposer.rs

```diff
@@ -403,15 +390,9 @@ impl Proposer {
     /// Whether we can advance the DAG or need to wait for the leader/more votes. This is only relevant in
     /// partial synchrony. Note that if we timeout, we ignore this check and advance anyway.
```

Comment:
nit: `This is only relevant in partial synchrony` --> nothing bad leaving this here, but maybe now this can be a bit confusing as we work by defacto in partial synchrony

Reply:
fixed